src/app/basket/page.tsx

```diff
@@ -32,12 +32,9 @@ export default function Page() {
 
   const requestNow = async () => {
     try {
-      await createApplication(basket.map((dataset) => dataset.id));
-      setAlert({
-        message: "Application created successfully",
-        type: "success",
-      });
+      const response = await createApplication(basket.map((dataset) => dataset.id));
       emptyBasket();
+      window.location.href = `/applications/${response.applicationId}`;
```

Comment:
We can use the redirect function from Next instead of manipulating directly the window object

Reply:
@admy7 
I tried redirect approach first but it was not working. Maybe we can take a look at it together tomorrow

Reply:
@inderps @admy7 as this is a tiny thing, I hope you don't mind that approve and merge this PR. Can you please align and open another PR later?

Reply:
Np